lib/fluent/plugin/monitoring.rb

```diff
@@ -97,21 +99,27 @@ def initialize(project_id, monitored_resource, gcm_service_address)
       require 'opencensus'
       require 'opencensus-stackdriver'
       @log = $log # rubocop:disable Style/GlobalVars
-      @recorder = OpenCensus::Stats.ensure_recorder
-      @exporter = OpenCensus::Stats::Exporters::Stackdriver.new(
-        project_id: project_id,
-        metric_prefix: 'agent.googleapis.com/agent',
-        resource_type: monitored_resource.type,
-        resource_labels: monitored_resource.labels,
-        gcm_service_address: gcm_service_address
-      )
+      @project_id = project_id
+      @monitored_resource = monitored_resource
+      @gcm_service_address = gcm_service_address
+      @recorders = { DEFAULT_PREFIX => OpenCensus::Stats.ensure_recorder }
```

Comment:
Can we just make @recorders and @exporters empty here?

Reply:
Good point. Removed.